src/alfrescoApiClient.js

```diff
@@ -205,6 +205,11 @@ class AlfrescoApiClient extends ApiClient {
         // set header parameters
         request.set(this.defaultHeaders).set(this.normalizeParams(headerParams));
 
+        // set the proxy user name if present
+        if (!!proxyUserName) {
```

Comment:
this.config.proxyUserName should works and at this point you don't need to  pass the param in the firm anymore